project/Makefile

```diff
@@ -14,7 +14,7 @@ lint-composer:
 .PHONY: lint-composer
 
 lint-yaml:
-	find . -name '*.yml' -not -path './vendor/*' -not -path './src/Resources/public/vendor/*' | xargs yaml-lint
+	yaml-lint -i -q -e vendor .
```

Comment:
Are there long counterparts for these long options? If yes, it would be great to use them.

Reply:
No problem. I just amended the commit to switch to their long options counterparts.